src/providers/kafka-service.ts

```diff
@@ -0,0 +1,97 @@
+import { Component } from '@angular/core'
+import { NavController } from 'ionic-angular'
+import { Injectable } from '@angular/core'
+import { Observable } from 'rxjs/Observable'
+import 'rxjs/add/operator/map'
+import { Http, Response } from '@angular/http'
+
+import * as KafkaRest from 'kafka-rest/lib/client'
+var Kafka = new KafkaRest({ 'url': 'https://radararmt.ddns.net/kafka' }) //change to production or development url 
```

Comment:
`var Kafka = ...` is outside the KafkaService class. I am assuming `new KafkaRest(...)` is a necessary setup step. Is it not possible to do this in the constructor of our service, instead of a global variable?